javascript/ql/src/Security/CWE-400/PrototypePollutionUtility.ql

```diff
@@ -113,11 +120,57 @@ class EntriesEnumeratedPropName extends EnumeratedPropName {
   }
 }
 
+/**
+ * Gets a function that enumerates object properties when invoked.
+ *
+ * Invocations takes the following form:
+ * ```js
+ * fn(obj, (value, key, o) => { ... })
+ * ```
+ */
+SourceNode propertyEnumerator() {
+  result = moduleImport("for-own") or
+  result = moduleImport("for-in") or
+  result = moduleMember("ramda", "forEachObjIndexed") or
+  result = LodashUnderscore::member("forEach") or
+  result = LodashUnderscore::member("each")
+}
+
+/**
+ * Property enumeration through the `for-own` or `for-in` package.
```

Comment:
This docstring needs to be more general now that `propertyEnumerator` includes many more cases than just `for-own` and `for-in`